### PR TITLE
Pipify project

### PR DESCRIPTION
**Pull request by benedikt-richter**
**Context**
The proximity matcher webservice shall be easy to use and to be installable via pip.

**Changes**
* Poetry is introduced 
* relevant scripts are added to be accessible via cli after installation

**How to test**

* Build Python wheel via 
```
 poetry build
```
* Install the wheel (e.g. in some test virtualenv) via 
```
pip install --force-reinstall $PATH_ON_YOUR_MACHINE/proximity_matcher_webservice/dist/proximity_matcher_webservice-0.0.1-py3-none-any.whl
```
* Try running the scripts from terminal. E.g. 
```
create-vpt-pickle -i /tmp/tlsh-hashes.txt -o /tmp/licenses-tlsh.pickle
```

### CODE REVIEW

**Comment by benedikt-richter** [comment]
Greetings from @maxhbr

**Comment by maxhbr** [comment]
potential next target: https://github.com/nix-community/poetry2nix

File: src/proximity_matcher_webservice/run_proximity_matcher_server.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 - Armijn Hemel
+# SPDX-FileCopyrightText: TNG Technology Consulting GmbH <https://www.tngtech.com>
```
Comment (armijnhemel) [inline src/proximity_matcher_webservice/run_proximity_matcher_server.py]: Not sure that this makes sense.

**Comment by armijnhemel** [comment]
Thanks for this PR. Apart from the copyright statement that doesn't make sense it looks good.

**Comment by benedikt-richter** [comment]
Thanks for the quick review! I modified the copyright.

File: src/proximity_matcher_webservice/run_proximity_matcher_server.py (hunk 1 above)
Comment (armijnhemel) [reply src/proximity_matcher_webservice/run_proximity_matcher_server.py]: > Thanks for the quick review! I modified the copyright.

yes, but not exactly what I meant! This is a new file, so not sure I should be mentioned in it. I also assume that the rights holder in this case would be TNG and not you personally. In that case the old format is much more correct (don't ask me, ask the German IT lawyers) ;-)

Comment (benedikt-richter) [reply src/proximity_matcher_webservice/run_proximity_matcher_server.py]: I see. The copyright is modified accordingly.

**Comment by benedikt-richter** [comment]
@armijnhemel are there some open points left to improve?

**Comment by armijnhemel** [comment]
> @armijnhemel are there some open points left to improve?

No, I just need to find time to test and then merge.

**Review by armijnhemel — APPROVED** [review]

**Pull request merged**